### ISSUE TITLE
Manager Version doesn't update 

### ISSUE THREAD

**Issue by YouSummoner** [revanced/revanced-manager#419]
### Type

Error while running the manager

### Bug description

The manager keeps downloading version 0.0.34 over and over after the latest 0.0.35 was released.
I also tried uninstalling and downloading the 0.0.35 update from GitHub, but it too actually shows 0.0.34 as the manager Version in settings and has the same issue with downloading the update.

### Steps to reproduce

1. Open manager
2. Update
3. Check version 

### Android version

Android 12.1

### Manager version

0.0.34

### Target package name

.

### Target package version.

.

### Installation type

Root

### Patches selected.

.

### Device logs (exported using Manager settings).

```shell
--------- beginning of main
10-20 05:35:26.090  6197  6197 E manager.flutter: Not starting debugger since process cannot load the jdwp agent.
10-20 05:35:26.091  6197  6197 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:35:26.110  6197  6197 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10443; state: ENABLED
10-20 05:35:26.202  6197  6197 V GraphicsEnvironment: ANGLE Developer option for 'app.revanced.manager.flutter' set to: 'default'
10-20 05:35:26.203  6197  6197 V GraphicsEnvironment: ANGLE GameManagerService for app.revanced.manager.flutter: false
10-20 05:35:26.268  6197  6197 W manager.flutter: type=1400 audit(0.0:3495): avc: denied { read } for name="max_map_count" dev="proc" ino=3988924 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=app.revanced.manager.flutter
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: QUALCOMM build                   : 5af5ff1, Iff02ff076a
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Build Date                       : 01/05/21
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.32.02.06
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Local Branch                     : mybrancheb4f33c0-fd8a-f1f2-ab18-5cb90179c3d8
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Remote Branch                    : quic/gfx-adreno.lnx.1.0.r114-rel
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Remote Branch                    : NONE
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Reconstruct Branch               : NOTHING
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Build Config                     : S P 10.0.7 AArch64
10-20 05:35:26.282  6197  6197 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
10-20 05:35:26.285  6197  6197 I AdrenoGLES-0: PFP: 0x016ee190, ME: 0x00000000
10-20 05:35:26.391  6197  6217 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:35:26.405  6197  6224 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:35:26.407  6197  6224 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:35:26.421  6197  6238 W SQLiteLog: (28) double-quoted string literal: "0B1FF185ABD2EC1F352F76FABC54C393"
10-20 05:35:26.422  6197  6238 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:35:26.469  6197  6224 I flutter : dynamic_color: Core palette detected.
10-20 05:35:26.518  6197  6225 I Gralloc4: mapper 4.x is not supported
10-20 05:35:26.518  6197  6225 W Gralloc3: mapper 3.x is not supported
10-20 05:35:26.521  6197  6238 W SQLiteLog: (28) double-quoted string literal: "8384255B7F11AA240121190919F1B9E3"
10-20 05:35:26.521  6197  6238 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:35:26.522  6197  6225 W Gralloc4: allocator 4.x is not supported
10-20 05:35:26.522  6197  6225 W Gralloc3: allocator 3.x is not supported
10-20 05:35:26.524  6197  6225 I Gralloc2: Adding additional valid usage bits: 0x0
10-20 05:35:26.546  6197  6225 E BLASTBufferQueue: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#1](f:0,a:0) rejecting buffer:active_size=1080x2340, requested_size=1080x2340 buffer{size=1080x2298 transform=0}
10-20 05:35:26.632  6254  6254 W sh      : type=1400 audit(0.0:3496): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:26.652  6254  6254 W sh      : type=1400 audit(0.0:3497): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:26.660  6254  6254 W sh      : type=1400 audit(0.0:3498): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:26.672  6254  6254 W sh      : type=1400 audit(0.0:3499): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.276  6254  6254 W sh      : type=1400 audit(0.0:3508): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.300  6254  6254 W sh      : type=1400 audit(0.0:3509): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.324  6254  6254 W sh      : type=1400 audit(0.0:3510): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.344  6254  6254 W sh      : type=1400 audit(0.0:3511): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.444  6254  6254 W sh      : type=1400 audit(0.0:3512): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:35:27.675  6197  6238 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:35:27.675  6197  6238 W SQLiteLog: (28) double-quoted string literal: "11B99312190D6CE22E285EFF67ABB046"
10-20 05:35:27.686  6197  6238 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:35:27.686  6197  6238 W SQLiteLog: (28) double-quoted string literal: "8A3FB4FCA9AC8DA8FFA6699D51729CB7"
10-20 05:38:47.290  6461  6461 E manager.flutter: Not starting debugger since process cannot load the jdwp agent.
10-20 05:38:47.292  6461  6461 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:38:47.317  6461  6461 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10443; state: ENABLED
10-20 05:38:47.372  6461  6461 V GraphicsEnvironment: ANGLE Developer option for 'app.revanced.manager.flutter' set to: 'default'
10-20 05:38:47.373  6461  6461 V GraphicsEnvironment: ANGLE GameManagerService for app.revanced.manager.flutter: false
10-20 05:38:47.452  6461  6461 W manager.flutter: type=1400 audit(0.0:3520): avc: denied { read } for name="max_map_count" dev="proc" ino=3988924 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=app.revanced.manager.flutter
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: QUALCOMM build                   : 5af5ff1, Iff02ff076a
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Build Date                       : 01/05/21
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.32.02.06
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Local Branch                     : mybrancheb4f33c0-fd8a-f1f2-ab18-5cb90179c3d8
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Remote Branch                    : quic/gfx-adreno.lnx.1.0.r114-rel
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Remote Branch                    : NONE
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Reconstruct Branch               : NOTHING
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Build Config                     : S P 10.0.7 AArch64
10-20 05:38:47.469  6461  6461 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
10-20 05:38:47.473  6461  6461 I AdrenoGLES-0: PFP: 0x016ee190, ME: 0x00000000
10-20 05:38:47.604  6461  6487 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:38:47.621  6461  6486 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:38:47.622  6461  6486 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:38:47.638  6461  6500 W SQLiteLog: (28) double-quoted string literal: "0B1FF185ABD2EC1F352F76FABC54C393"
10-20 05:38:47.638  6461  6500 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:38:47.689  6461  6486 I flutter : dynamic_color: Core palette detected.
10-20 05:38:47.738  6461  6487 I Gralloc4: mapper 4.x is not supported
10-20 05:38:47.741  6461  6487 W Gralloc3: mapper 3.x is not supported
10-20 05:38:47.742  6461  6500 W SQLiteLog: (28) double-quoted string literal: "8384255B7F11AA240121190919F1B9E3"
10-20 05:38:47.742  6461  6500 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:38:47.747  6461  6487 W Gralloc4: allocator 4.x is not supported
10-20 05:38:47.748  6461  6487 W Gralloc3: allocator 3.x is not supported
10-20 05:38:47.752  6461  6487 I Gralloc2: Adding additional valid usage bits: 0x0
10-20 05:38:47.784  6461  6487 E BLASTBufferQueue: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#1](f:0,a:0) rejecting buffer:active_size=1080x2340, requested_size=1080x2340 buffer{size=1080x2298 transform=0}
10-20 05:38:47.832  6511  6511 W sh      : type=1400 audit(0.0:3521): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:47.852  6511  6511 W sh      : type=1400 audit(0.0:3522): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:47.872  6511  6511 W sh      : type=1400 audit(0.0:3523): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:47.896  6511  6511 W sh      : type=1400 audit(0.0:3524): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:48.836  6511  6511 W sh      : type=1400 audit(0.0:3529): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:48.856  6511  6511 W sh      : type=1400 audit(0.0:3530): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:48.872  6511  6511 W sh      : type=1400 audit(0.0:3531): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:48.896  6511  6511 W sh      : type=1400 audit(0.0:3532): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:48.924  6511  6511 W sh      : type=1400 audit(0.0:3533): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:38:49.406  6461  6500 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:38:49.407  6461  6500 W SQLiteLog: (28) double-quoted string literal: "11B99312190D6CE22E285EFF67ABB046"
10-20 05:38:49.416  6461  6500 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:38:49.416  6461  6500 W SQLiteLog: (28) double-quoted string literal: "8A3FB4FCA9AC8DA8FFA6699D51729CB7"
10-20 05:38:54.710  6461  6478 I OpenGLRenderer: Davey! duration=6952ms; Flags=1, FrameTimelineVsyncId=3270983, IntendedVsync=54560737415272, Vsync=54561004081928, InputEventId=0, HandleInputStart=54561016273247, AnimationStart=54561016276060, PerformTraversalsStart=54561016277206, DrawStart=54561017774393, FrameDeadline=54560774581938, FrameInterval=54561016263664, FrameStartTime=16666666, SyncQueued=54561018448352, SyncStart=54561018611008, IssueDrawCommandsStart=54561020284237, SwapBuffers=54561027356164, FrameCompleted=54567690012255, DequeueBufferDuration=4823229, QueueBufferDuration=2577083, GpuCompleted=54567690012255, SwapBuffersCompleted=54561031049237, DisplayPresentTime=0, 
10-20 05:39:01.026  6461  6461 D CompatibilityChangeReporter: Compat change id reported: 147798919; UID 10443; state: ENABLED
10-20 05:39:16.048  6699  6699 E manager.flutter: Not starting debugger since process cannot load the jdwp agent.
10-20 05:39:16.050  6699  6699 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:39:16.081  6699  6699 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10443; state: ENABLED
10-20 05:39:16.134  6699  6699 V GraphicsEnvironment: ANGLE Developer option for 'app.revanced.manager.flutter' set to: 'default'
10-20 05:39:16.134  6699  6699 V GraphicsEnvironment: ANGLE GameManagerService for app.revanced.manager.flutter: false
10-20 05:39:16.212  6699  6699 W manager.flutter: type=1400 audit(0.0:3545): avc: denied { read } for name="max_map_count" dev="proc" ino=3988924 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=app.revanced.manager.flutter
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: QUALCOMM build                   : 5af5ff1, Iff02ff076a
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Build Date                       : 01/05/21
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.32.02.06
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Local Branch                     : mybrancheb4f33c0-fd8a-f1f2-ab18-5cb90179c3d8
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Remote Branch                    : quic/gfx-adreno.lnx.1.0.r114-rel
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Remote Branch                    : NONE
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Reconstruct Branch               : NOTHING
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Build Config                     : S P 10.0.7 AArch64
10-20 05:39:16.230  6699  6699 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
10-20 05:39:16.234  6699  6699 I AdrenoGLES-0: PFP: 0x016ee190, ME: 0x00000000
10-20 05:39:16.369  6699  6718 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
10-20 05:39:16.398  6699  6723 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:39:16.399  6699  6723 I flutter : ReVanced API: Using CronetEngine + true
10-20 05:39:16.417  6699  6738 W SQLiteLog: (28) double-quoted string literal: "0B1FF185ABD2EC1F352F76FABC54C393"
10-20 05:39:16.417  6699  6738 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:39:16.504  6699  6723 I flutter : dynamic_color: Core palette detected.
10-20 05:39:16.577  6699  6724 I Gralloc4: mapper 4.x is not supported
10-20 05:39:16.579  6699  6724 W Gralloc3: mapper 3.x is not supported
10-20 05:39:16.581  6699  6738 W SQLiteLog: (28) double-quoted string literal: "8384255B7F11AA240121190919F1B9E3"
10-20 05:39:16.582  6699  6738 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
10-20 05:39:16.583  6699  6724 W Gralloc4: allocator 4.x is not supported
10-20 05:39:16.584  6699  6724 W Gralloc3: allocator 3.x is not supported
10-20 05:39:16.586  6699  6724 I Gralloc2: Adding additional valid usage bits: 0x0
10-20 05:39:16.622  6699  6724 E BLASTBufferQueue: [SurfaceView[app.revanced.manager.flutter/app.revanced.manager.flutter.MainActivity]#1](f:0,a:0) rejecting buffer:active_size=1080x2340, requested_size=1080x2340 buffer{size=1080x2298 transform=0}
10-20 05:39:16.660  6749  6749 W sh      : type=1400 audit(0.0:3546): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:16.684  6749  6749 W sh      : type=1400 audit(0.0:3547): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:16.704  6749  6749 W sh      : type=1400 audit(0.0:3548): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:16.728  6749  6749 W sh      : type=1400 audit(0.0:3549): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.336  6749  6749 W sh      : type=1400 audit(0.0:3554): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.360  6749  6749 W sh      : type=1400 audit(0.0:3555): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.400  6749  6749 W sh      : type=1400 audit(0.0:3556): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.424  6749  6749 W sh      : type=1400 audit(0.0:3557): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.464  6749  6749 W sh      : type=1400 audit(0.0:3558): avc: denied { read } for name="/" dev="sda11" ino=2 scontext=u:r:untrusted_app:s0:c187,c257,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
10-20 05:39:17.951  6699  6738 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:39:17.952  6699  6738 W SQLiteLog: (28) double-quoted string literal: "11B99312190D6CE22E285EFF67ABB046"
10-20 05:39:17.962  6699  6738 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
10-20 05:39:17.962  6699  6738 W SQLiteLog: (28) double-quoted string literal: "8A3FB4FCA9AC8DA8FFA6699D51729CB7"
```

### Screenshots or videos

![Screenshot_20221020-053245_Material Files](https://user-images.githubusercontent.com/107224555/196827309-0e00c825-7bbf-4996-84fd-be68d655965a.png)
![Screenshot_20221020-053250_Package installer](https://user-images.githubusercontent.com/107224555/196827318-c5c6e345-a292-4522-98e1-93229f18bce8.png)
![Screenshot_20221020-053335_ReVanced Manager](https://user-images.githubusercontent.com/107224555/196827324-606475ca-a4c0-457f-b4d6-97f17b9616ce.png)

### Solution

None

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager

**Comment by YouSummoner** [comment]
The patch update info button seems to be working in this latest release, which wasn't in the previous one.
It seems like the manager is actually updating but it doesn't really know that it's on version 0.0.35.

**Comment by Shished** [comment]
.35 APK has .34 version string.

**Comment by mhetralla** [comment]
They forgot to bump the version to 0.0.35, so the updater can't differentiate between 0.0.34 and this "0.0.35". It'll be fixed probably on 0.0.36.

**Comment by Ushie** [comment]
😅
